docs/book/src/migration/migration_guide_gov3_to_gov4.md

```diff
@@ -152,6 +171,16 @@ Change the image name in the Makefile if needed.
 Finally, we can run `make` and `make docker-build` to ensure things are working
 fine.
 
+<aside class="note">
```

Comment:
I personally feels it might not necessary considering the case where users could gain experience already when working with v3.

Reply:
+1